Login.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
+        window.location.href = "/home";
```

Comment:
Avoid using 'window.location.href' for navigation. It causes the browser to refresh and the state of your app will be lost. Consider using the 'react-router-dom' library instead.